dockstore-webservice/src/main/resources/import.sql

```diff
@@ -57,4 +57,9 @@ CREATE UNIQUE INDEX one_token_link_per_identify2 ON token USING btree (username,
 
 create unique index collection_displayname_index on collection (LOWER(displayname), organizationid);
 
-create unique index collection_categoryname_index on collection (LOWER(name)) where dtype = 'Category';
+DROP INDEX collection_name_index;
```

Comment:
As commented in the other PR, instead of

1. Create index x
2. Drop index x
3. Create index x differently than in step 1.

You should just do 3 alone. You're doing this pattern for 2 indexes. This is not a migrations file, where the history is important; this is just a file to reconcile Hibernate generated DB artifacts (as was pointed out to me)